src/utilities/scripts/aws_star.py

```diff
@@ -7,26 +7,43 @@
 
 
 def main():
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(description="Create a shell to run alignment jobs" + 
+                                     " with STAR and htseq for multiple samples all together")
 
     parser.add_argument(
-        "--branch", default="master", help="branch of utilities repo to use"
+        "--branch", default="master",
+        help="Branch of utilities repo to use"
     )
 
-    parser.add_argument("taxon", choices=list(reference_genomes.keys()))
-    parser.add_argument("num_partitions", type=int)
     parser.add_argument(
-        "s3_input_path", help="The folder containing fastq.gz files to align"
+        "--taxon", choices=list(reference_genomes.keys()), required=True,
+        help="(required) Reference genome for the alignment run, " +
+        "selected from the reference_genomes dictionary keys from " +
+        "alignment.run_star_and_htseq.py"
```

Comment:
For multi-line strings it's preferable to do this (no `+`, a little more efficient, not that it matters here but it's good practice)
```suggestion
        help=(
            "(required) Reference genome for the alignment run, "
            "selected from the reference_genomes dictionary keys from "
            "alignment.run_star_and_htseq.py"
        )
```